### PR TITLE
Email change test improvements

### PR DESCRIPTION
**Pull request by individual-it**
small improvements to the testing code

### CODE REVIEW

File: tests/acceptance/features/apiProvisioningLDAP/users.feature [hunk 1]
```diff
@@ -118,8 +118,10 @@ Feature: Manage users using the Provisioning API
     And the administrator changes the email of user "user1" to "apiuser@example.com" using the provisioning API
     Then the OCS status code should be "<ocs-status-code>"
     And the HTTP status code should be "<http-status-code>"
-    # And the email address of user "user1" should be "ldapuser@example.com"
+    # And the email address of user "user1" should be "ldapuser@oc.com"
     And the email address of user "user1" should be "apiuser@example.com"
+    And the LDAP users are resynced
+    And the email address of user "user1" should be "ldapuser@oc.com"
```
Comment (skshetry) [inline tests/acceptance/features/apiProvisioningLDAP/users.feature]: Was this fixed?

Comment (individual-it) [reply tests/acceptance/features/apiProvisioningLDAP/users.feature]: was it broken?
the admin still can overwrite the setting https://github.com/owncloud/core/issues/33186 but when synced the correct LDAP setting will come back

Comment (skshetry) [reply tests/acceptance/features/apiProvisioningLDAP/users.feature]: No, just asked as it was commented before.

**Comment by phil-davis** [comment]
drone apiAuth job was cancelled after running for 3 hours.
I restarted it.
We need to see why that suite (from core) is taking so long to run nowadays.

**Pull request merged**